src-docs/src/views/super_select/super_select_example.js

```diff
@@ -54,6 +59,23 @@ const superSelectStatesSnippet = `<EuiSuperSelect
 
 export const SuperSelectExample = {
   title: 'Super select',
+  intro: (
+    <EuiCallOut
+      iconType="questionInCircle"
+      title="Learn more about EUI selection components"
+    >
+      <p>
+        See{' '}
+        <EuiLink
+          href="https://github.com/elastic/eui/discussions/7049"
+          target="_blank"
+        >
+          EUI's in-depth guide on which selection component to use{' '}
```

Comment:
If we tweak the title above, "to use" ends up feeling a little redundant here
```suggestion
          EUI's in-depth guide to selection components{' '}
```